shell/platform/embedder/embedder.h

```diff
@@ -661,13 +679,55 @@ typedef struct {
   VoidCallback destruction_callback;
 } FlutterSoftwareBackingStore;
 
+typedef enum {
```

Comment:
Could you keep enum value naming convention consistent? ie: kFlutterPlatformViewMutationType*

Reply:
Yeah, my bad. Fixed.